gateway/modules/schema/inspection.go

```diff
@@ -99,6 +88,18 @@ func generateInspection(dbType, col string, fields []model.InspectorFieldType, f
 			fieldDetails.IsPrimary = true
 		}
 
+		// Set auto increment
+		fieldDetails.AutoIncrementInfo = new(model.AutoIncrementInfo)
+		if field.AutoIncrement == "true" {
+			fieldDetails.AutoIncrementInfo = &model.AutoIncrementInfo{IsEnabled: true}
```

Comment:
What about start from and increment by?

Reply:
I would suggest you remove support for increment by and start from since its available in sqlserver only. In that case this piece of code is right. You'll have to change the part where you parse the schema